agent/functional_tests/testdata/taskdefinitions/oom-task/task-definition.json

```diff
@@ -1,12 +1,12 @@
 {
   "family": "ecsftest-oom-task",
-  "memory": "64",
+  "memory": "10",
```

Comment:
Why decreasing memory can help test pass? Is it because a lot of tests failure are caused by OOM?